tests/test_template/test_spelling_gate.py

```
@@ -0,0 +1,227 @@
+    tmp_path: Path, copier: CopierFixture
+) -> None:
+    """The generated typos.toml parses and matches the generator output."""
+    project = render_project(
+        tmp_path,
+        copier,
+        project_name="SpellingDrift",
+        package_name="spelling_drift",
+    )
+
+    subprocess.run(  # noqa: S603 - runs the rendered, trusted generator.
+        ["uv", "run", "scripts/generate_typos_config.py"],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)